src/Ledger/constants.js

```diff
@@ -36,12 +36,12 @@ export const LEDGER_STATUS_OPTIONS = Object.values(LEDGER_STATUS).map(status =>
 }));
 
 export const ADD_AVAILABLE_TO = {
-  available: 'Available',
+  transfer: 'Available',
```

Comment:
should it be `transfer: 'Transfer'`?

Reply:
according to this - no https://github.com/folio-org/acq-models/blob/master/mod-finance/schemas/budget_rollover.json#L30